app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckEoriDetailsController.scala

```diff
@@ -137,16 +136,13 @@ class CheckEoriDetailsController @Inject() (
     }
   }
 
-  private def withMovementReferenceNumberAnswer(
-    f: (SessionData, FillingOutClaim, MovementReferenceNumberAnswer) => Future[Result]
+  private def withSessionData(
+    f: (SessionData, FillingOutClaim) => Future[Result]
   )(implicit request: RequestWithSessionData[_]): Future[Result] =
     request.sessionData.flatMap(s => s.journeyStatus.map(s -> _)) match {
-      case Some((sessionData, fillingOutClaim @ FillingOutClaim(_, _, draftClaim: DraftClaim))) =>
-        val maybeMovementReferenceNumberAnswers = draftClaim.fold(_.movementReferenceNumberAnswer)
-        maybeMovementReferenceNumberAnswers.fold[Future[Result]](
-          f(sessionData, fillingOutClaim, IncompleteMovementReferenceNumberAnswer.empty)
-        )(f(sessionData, fillingOutClaim, _))
-      case _                                                                                    => Future.successful(Redirect(baseRoutes.StartController.start()))
+      case Some((sessionData, fillingOutClaim @ FillingOutClaim(_, _, _: DraftClaim))) =>
+        f(sessionData, fillingOutClaim)
+      case _                                                                           => Future.successful(Redirect(baseRoutes.StartController.start()))
```

Comment:
I think this was an unnecessary C&P by @aneksamun when he was moving the email checking code across. Not a central part of this PR.

Reply:
sorry I don't really understand what is wrong with this?